src/components/Atoms/Input/index.jsx

```diff
@@ -0,0 +1,36 @@
+import React, { useRef, useEffect } from "react";
+import styled from "styled-components";
+import COLOR from "../../../variables/color";
+import TEXT from "../../../variables/texts";
+
+const Input = (props) => {

```

Comment:
```suggestion
const Input = ({defaultValue, onEditComplete}) => {
```
分割代入でPropsを受け取ると、どのPropsを受け取ることを想定しているのかが一目でわかるため使う側にとってありがたくなりそうです。